app/views/mailer/reply.html.erb

```diff
@@ -18,6 +18,9 @@
   </div>
 
   <p style="font-family: 'Open Sans','Helvetica Neue',arial,sans-serif;font-size: 12px;font-weight: normal;line-height: 20px;">
-    <%= t("mailers.config.manage_email_subscriptions") %> <%= link_to t("account.show.title"), account_url, style: "color: #2895F1; text-decoration:none;" %>
+    <%= t("mailers.config.manage_email_subscriptions") %>
+    <%= link_to t("mailers.reply.unsubscribe"),
+                edit_subscriptions_url(token: @receiver.subscriptions_token),
+                style: "color: #2895F1; text-decoration:none;" %>
```

Comment:
Since we're changing these lines, how about using just one translation, like in other places?

```suggestion
    <%= sanitize(t("...", unsubscribe: link_to(...))) %>
```

This way it'll be easier to translate this sentece to languages where the syntax has nothing to do with the one used in English.